composer.json

```diff
@@ -23,14 +23,15 @@
     },
     "require": {
         "php": ">=7.1.3",
+        "ext-dom": "*",
         "jkphl/dom-factory": "^1",
         "jkphl/rdfa-lite-microdata": "^0.4.4",
-        "league/uri": "^5.0",
-        "mf2/mf2": "^0.4",
+        "league/uri": "^6.5",
+        "mf2/mf2": "^0.5",
```

Comment:
This bump already is supported by the previous clause. Did 0.5 introduce a feature micrometa will have to support?

Reply:
Possibly, https://github.com/microformats/php-mf2/releases/tag/v0.5.0
the only change here that sounds potentially problematic is the url parsing change. I'll just drop this back to 0.4 for now, this should go through separate review

Reply:
Updated